main.css

```diff
@@ -1 +1,38 @@
+.allBoxes{
+  display:flex;
+  flex-flow: row wrap;
+  border: 2px solid black;
+  border-radius: 10%;
+  width: 450px;
+  height: 560px;
+  padding: 2px;
+  margin: 0;
+  justify-content: space-around;
+  background-color:red;
+}
+
+.answer{
+  background-color:grey;
+  border-radius: 10%;
+  width:400px;
+  height: 100px;
+}
+.box{
+  background-color:white;
+  width:100px;
+  height: 100px;
+  border: 3px solid black;
+  border-radius: 25%;
+  line-height:100px;
+  text-align: center;
+  font-size: 36px;
+}
+
+
+#operations{
+  background-color:yellow;
+}
+/* .aside{
+  border: 2px solid white;
+} */
```

Comment:
Great start on this Nichole! I've seen your calculators with some JavaScript functionality. I love the creativity and thought you put into the calculator designs. Also love how persistent you are in bringing all the pieces together. Great job!